conda/cli/install.py

```diff
@@ -231,7 +231,7 @@ def install(args, parser, command='install'):
         Please remove duplicates of %s package""" % name
                 raise CondaCorruptEnvironmentError(msg)
 
-            pkgs = sorted(r.get_pkgs(name))
+            pkgs = r.get_pkgs(name)
```

Comment:
`get_pkgs` automatically sorts now. Since the proper sorting is defined within `Resolve` anyway, and most uses of `get_pkgs` were sorting anyway.